### ISSUE TITLE
Misleading of Guest accounts

### ISSUE THREAD

**Issue by eric-ycw** [ironbane/IronbaneServerLegacy#348]
Many guests(new players that have started playing the game) were on steady progress, already have pretty good weapons and amour from chests was very disappointed when they needed a register a new account and all progress are  gone. We should add something to tell the players that if playing as guest, progress won't be saved.

**Comment by nickjanssen** [comment]
Yes, we should add a feature in the future where you can convert a guest account to a registered one.

**Comment by ghost** [comment]
+1 on the conversion process. Would it be possible to save some user data temporarily to the server for, say, 24 hours, and assign a cookie to the user's browser with an ID to to give to the server on the next play so they have a little bit more time to register?

That way if they decide they do wanna keep playing, even if they've or accidentally closed the browser or crashed, it's not a "Whoops you're done!" kind of deal. Especially if the server does crash, or restart. It's kind of "meh..." at that point for guests.

**Comment by bsparks** [comment]
it's actually supposed to be keeping the guest id as a cookie and using the same guest every time (until cookie expiry) this is sort of a bug. also I think that adding stuff like facebook and twitter login will make it "easier" to register and also help with retention

**Comment by ghost** [comment]
Social networking connections are a definite plus. Google+ as well for those hipsters (and by hipsters I mean they are using something with currently little to no popularity [I kid!]).
